src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -23,6 +23,7 @@ public static class StringTests
         [InlineData(new char[] { 'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', '\0', 'i', 'j' }, "abcdefgh")]
         [InlineData(new char[] { 'a', '\0' }, "a")]
         [InlineData(new char[] { '\0' }, "")]
+        [InlineData(new char[] { '?', '@', ' ', '\0' }, "?@ ")] // ? and @ don't have overlapping bits
```

Comment:
@stephentoub I was waiting for you to comment on the CoreCLR pull request, since I've switched it to use an implementation not based on glibc's and reopened it. Then I can re-add a test case for the new implementation in this pull request.